python/ray/tests/test_actor.py

```diff
@@ -1393,6 +1393,27 @@ def get_actor(actor):
     assert origin == remote
 
 
+@pytest.mark.skipif(
+    sys.version_info < (3, 9), reason="GenericAlias is new in Python 3.9"
+)
+def test_classmethod_genericalias():
+    """
+    The Python built-in function `inspect.signature` doesn't support
+    classmethod(GenericAlias). Hence, if we call `inspect.signature(MyClass)`,
```

Comment:
It's not `inspect.signature(MyClass)` but `inspect.signature(__class_getitem__)`?

Reply:
Good catch. You are correct.